.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+version: 2.1
+jobs:
+  java:
+    machine:
+      image: circleci/classic:latest
+    working_directory: ~/misk
+    steps:
+      - checkout # check out source code to working directory
+      - run:
+          name: "Download and extract OpenJDK 11"
+          command: "mkdir -p ~/openjdk-11 && cd ~/openjdk-11 && curl -sSL https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x64_bin.tar.gz | tar -xz --strip-components 1"
+      - run:
+          name: "Set environment variables"
+          command: |
+            echo '
+              export PATH="$JAVA_HOME/bin:$PATH"
+              export JAVA_HOME="$HOME/openjdk-11"
+              export ENVIRONMENT=TESTING
+              # Not sure if this actually gets applied on the daemon
+              export GRADLE_OPTS=-Xmx4096m
+            ' >> $BASH_ENV
+      - run:
+          name: "Dump environment variables"
+          command: env
+      - run:
+          name: "Start MySQL"
+          # There's a race condition here, where it's possible that MySQL hasn't finished starting
+          # up before we try to connect to it. But Kotlin builds are so damn slow it'll never
+          # happen in practice.
+          command: docker run -d -p 3306:3306 --name mysql -e MYSQL_ALLOW_EMPTY_PASSWORD=1 mysql:5.7
+      - run:
+          name: "Build and test"
+          command: ./gradlew test -i --scan --parallel --build-cache
+  node:
+    working_directory: ~/misk-web
```

Comment:
Shouldn't this still be `~/misk`? This step is to build the `misk-web` tabs that are in Misk repo (config, loader, web actions)

Reply:
I checked the build logs and the tabs seemed to be built but it's unintuitive to me why that would be happening in the `~/misk-web` directory.

Reply:
It doesn't matter. `checkout` loads your code into `working_directory`. I copied this from wesley's old branch, but I can change it to `misk` and it'll work the same.